src/components/Layout/Layout.jsx

```diff
@@ -1,7 +1,35 @@
 import "./Layout.scss";
+import Menu from "../Menu/Menu";
+import MenuIcon from "../MenuIcon/MenuIcon";
+import { useState } from "react";
 
-const Layout = ({ children }) => {
-  return <div className="layout">{children}</div>;
-};
+const Layout = ({ children, isWithMenu }) => {
+  if (!isWithMenu) {
+    return <div className="layout">{children}</div>;
+  }
+
+  //Layout with Menu Icon
+  const [showMenu, setShowMenu] = useState(false);
 
+  const toggleMenu = () => {
+    setShowMenu(!showMenu);
+  };
+
+  return (
+    <>
```

Comment:
We don't need a fragment here anymore, as the div is the parent element

Reply:
deleted